doc/release/upcoming_changes/24126.change.rst

```diff
@@ -0,0 +1,15 @@
+Add mean keyword to var and std function
+----------------------------------------
+
+Often when the standard deviation is needed the mean is also needed; the same holds for the variance and the mean. With the current code the mean is then calculated twice, the change introduced here for the var and std functions allows for passing in a precalculated mean as an keyword argument.
```

Comment:
Can you reformat to ~80 columns? Also: no need to make the release note extensive, it can reference the docstring and just be a few lines. The examples should move to the docstring and be more complete, like the other examples.

Reply:
Moved and adjusted the example to the doc-string and reflowed the release note.